azure/.gitignore

```diff
@@ -0,0 +1,12 @@
+# Compiled TypeScript and CSS
```

Comment:
why does this directory need it's only .gitignore?  Most (all?) of these are already covered by the root one?

Reply:
Regarding this and the license file: I was operating under the "every monorepo could be airlifted out and 'just work'" philosophy, hence I copied gitignore etc. like the Routerlicious monorepo does. But you're right that it's not needed, so I can remove it if it's clearer. My philosophy could be outdated. :)

Reply:
I removed the LICENSE file but left the gitignore based on the airlifting philosophy.